src/applications/letters/tests/containers/LettersApp.unit.spec.jsx

```diff
@@ -58,5 +58,27 @@ describe('<LettersApp>', () => {
       );
       expect(text).to.not.contain('Rendered!');
     });
+
+    it('should render loading indicator if feature toggles are not available', () => {
+      const tree = SkinDeep.shallowRender(
+        <AppContent featureFlagsLoading>
+          <span>Rendered!</span>
+        </AppContent>,
+      );
+
+      expect(tree.subTree('va-loading-indicator')).to.exist;
+      expect(tree.text()).to.not.contain('Rendered');
+    });
+
+    it('should render children if feature toggles are available', () => {
+      const tree = SkinDeep.shallowRender(
+        <AppContent featureFlagsLoading={false}>
+          <span>Rendered!</span>
+        </AppContent>,
+      );
+
+      expect(tree.subTree('va-loading-indicator')).to.equal(false);
+      expect(tree.text()).to.contain('Rendered');
+    });
```

Comment:
Just a consideration, but we are trying to avoid SkinDeep when possible. Since you already have the data-testid setup you could probably use react-testing-library to write your tests like this:
```it('should render children if feature toggles are available', () => {
    const screen = render(
      <AppContent featureFlagsLoading={false}>
        <div data-testid="children" />
      </AppContent>,
    );

    expect(screen.queryByTestId('feature-flags-loading')).to.not.exist;
    expect(screen.queryByTestId('children')).to.exist;
  });

Reply:
Thanks! This was a straight-up copy/paste. I can fix this to use react-testing-library.